src/hooks/useSupplyWithdrawModal/Modal/index.tsx

```diff
@@ -260,6 +269,12 @@ const SupplyWithdrawModal: React.FC<SupplyWithdrawUiProps> = props => {
 
   const isWithdrawLoading = isRedeemLoading || isRedeemUnderlyingLoading;
 
+  // Hide modal while loading
+  if (!asset) {
+    // TODO: handle loading state (see https://app.clickup.com/t/2d4rcee)
+    return null;
```

Comment:
Could we return the loader here instead?

Reply:
Definitely